src/search/DownwardFiles.cmake

```diff
@@ -708,6 +709,7 @@ fast_downward_plugin(
     HELP "Plugin containing the code for operator counting heuristics"
     SOURCES
         operator_counting/constraint_generator
+        operator_counting/delete_relaxation_constraints
```

Comment:
Perhaps this was discussed before, but I'm not sure I understand why this a single all-or-nothing plugin rather than being able to plug in different constraints selectively. Perhaps we do this similarly for merge-and-shrink and PDBs? In any case, unrelated to this issue, just something one notices when seeing this change.

Reply:
This is even more true if we consider the dependencies of this plug-in on LM-Cut and PDBs , which are presumably only needed for some of the constraints.

Reply:
I created https://issues.fast-downward.org/issue1046 for this. I don't think there is a specific reason why it is the way it is. We could just as well change it.